test/sim/team-validator.js

```diff
@@ -486,6 +486,14 @@ describe('Team Validator', function () {
 		assert(illegal);
 	});
 
+	it('should reject exclusive G-Max moves added directly to a Pokemon\'s moveset', function () {
+		let team = [
+			{species: 'charizard-gmax', ability: 'blaze', moves: ['gmaxwildfire'], evs: {hp: 1}},
+		];
+		let illegal = TeamValidator.get('gen8anythinggoes').validateTeam(team);
+		assert(illegal);
```

Comment:
```suggestion
		let illegal = TeamValidator.get('gen8anythinggoes').validateTeam(team);
		assert(illegal);
		illegal = TeamValidator.get('gen8balancedhackmons').validateTeam(team);
		assert(illegal);
```

Reply:
I would probably future-proof it and use ``gen8customgame@@@-nonexistent`` because gen 8 balanced hackmons might not exist on PS in some generation in the distant future.